includes/emails/class-email.php

```diff
@@ -83,6 +83,32 @@ public static function init( $meta = [] ) {
 			$meta = apply_filters( 'hivepress/v1/emails/' . hp\get_class_name( $class ) . '/meta', $meta );
 		}
 
+		foreach ( hp\get_array_value( $meta, 'tokens', [] ) as $token ) {
+
+			// Get class object.
+			$class = hp\create_class_instance( 'HivePress\Models\\' . $token );
```

Comment:
It seems that we have to be careful not naming tokens as any of the existing models, maybe lets add extra config to token meta (to explicitly mark some tokens as object-like), like we did for models in Attribute? Otherwise someone may define a token like "request" and it will conflict with Requests.